.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,35 +1,41 @@
 <!--
-    If you have any questions regarding your scenario or components behavior, please provide a detailed explanation.
-    If you own an active DevExtreme license, feel free to contact us in the Support Center (https://www.devexpress.com/ask) to receive an answer shortly.
+    We cannot guarantee a quick answer on GitHub.
+    If you own an active DevExtreme license, feel free to contact us in the Support Center (https://www.devexpress.com/ask)
+    to receive an answer shortly.
 -->
 
 <!--
-    Checked checkbox should look like this: [x]
+    A checked check box should look like this: [x]
 -->
 - [ ] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.
```

Comment:
```suggestion
- [ ] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) for duplicates.
```

Reply:
I read it as "I've looked through all issues to find if any of them appears more than once". What we want the creator of an issue to do is to search the issues to find if there is already one that reports the problem the creator has.

Reply:
```suggestion
- [ ] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and did not find duplicates.
```

Reply:
I'm afraid this version has the same issue.